src/redturtle/prenotazioni/content/prenotazioni_folder.py

```diff
@@ -209,6 +209,26 @@ def notify_on_refuse_message_default_factory(context):
     )
 
 
+@provider(IContextAwareDefaultFactory)
+def notify_as_reminder_subject_default_factory(context):
+    return getattr(context, "translate", translate)(
+        _(
+            "notify_as_reminder_subject_default_value",
+            "You have the upcomming booking on ${booking_date}",
```

Comment:
typo.

But probably could be better something like "Reminder for upcoming booking on ${booking_date}"

Reply:
Solved